Azure-Samples/msdocs-django-postgresql-sample-app
# Add support for Azure Developer CLI and related functionality

## Purpose

As discussed over email, this PR adds support for the Azure Dev CLI. That entails adding infra/ folder with Bicep files, azure.yaml file in the root, .devcontainer for Codespaces support, .github with azd workflow, and more details in the readme.

I also renamed .env.example to .env.sample since we made that change in the Flask app to match the docs, and I added .env.sample.devcontainer to provide devcontainer-specific values.

None of the code has changed.

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[X] Refactoring (no functional changes, no api changes)
[X] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

Follow the instructions in the Readme inside Github Codespaces. You should be able to run locally inside Codespaces and deploy from there. Here's a video of me deploying this app:
https://youtu.be/JDlZ4TgPKYc

Here's a video of me running the Flask app (similar process to this one):
https://youtu.be/vEtR5qsXfZE
